person.rb

```diff
@@ -2,14 +2,15 @@
 require_relative 'decorator'
 
 class Person < Namebale
```

Comment:
Great job @boniface193 🤝

- [ ] As stated in the project requirements, kindly ensure the method for adding rentals should be present. 
![add-rental person](https://github.com/boniface193/oop_ruby/assets/63917725/9c937044-0b66-42f8-8849-04067a3523c6)

Reply:
is the comment above different from this?

Reply:
Hello @boniface193 ✋. 
Yes, the comment is different. The references are on different classes.